main/inc/lib/notification.lib.php

```diff
@@ -384,6 +391,18 @@ public function formatContent($messageId, $content, $senderInfo)
             }
         }
 
+        $accessConfig = [];
+        $useMultipleUrl = api_get_configuration_value('multiple_access_urls');
+        if ($useMultipleUrl && $checkUrls) {
+            $accessUrls = api_get_access_url_from_user($senderInfo['user_info']['user_id']);
```

Comment:
It should be $senderInfo['user_id'] instead of $senderInfo['user_info']['user_id'] because $senderInfo['user_info'] does not exist